### PR TITLE
feat(microservices): added rabbitmq queue consumer options

### PR DESCRIPTION
**Pull request by webjoaoneto**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
This PR introduces a new configuration for RabbitMQ microservices queue.

Adds an option to configure consumer arguments

```typescript
export const RabbitMQQueueNest = {
  name: 'QUEUE_STREAM',
  transport: Transport.RMQ,
  options: {
    urls: [rabbitMqUrl],
    queue: RabbitMqQueues.USUARIOS.name,
    noAck: false,
    prefetchCount: 1000,
    consumerOptions: {
      arguments: {
        'x-stream-offset': '7D',
      },
    },
    queueOptions: {
      durable: true,
      deadLetterExchange: ``,
      deadLetterRoutingKey: RabbitMqQueues.USUARIOS.dlq,
      arguments: {
        'x-queue-type': 'stream',
      },
    },
  },
};

```

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?
Actually, only `noAck` argument is passed to the consumerOptions, but rabbitmq accept any argument from options that changes the way that rabbitmq consumer works

## What is the new behavior?
This PR enables the possibility to add any argument to the RabbitMQ Consumer

## Does this PR introduce a breaking change?
- [ ] Yes
- [X] No

## Other information
This is not a break change, only introduces an new option to rabbitmq queue configuration

### CODE REVIEW

File: packages/microservices/server/server-rmq.ts [hunk 1]
```diff
@@ -43,6 +44,7 @@ export class ServerRMQ extends Server implements CustomTransportStrategy {
   protected readonly queue: string;
   protected readonly prefetchCount: number;
   protected readonly queueOptions: any;
+  protected readonly consumerOptions: any;
```
Comment (kamilmysliwiec) [inline packages/microservices/server/server-rmq.ts:47]: we should be able to use `AmqplibConsumerOptions` instead of `any` here

Comment (webjoaoneto) [reply packages/microservices/server/server-rmq.ts:47]: i just keep the style of code that aready exists.

File: packages/microservices/server/server-rmq.ts [hunk 2]
```diff
@@ -152,6 +157,7 @@ export class ServerRMQ extends Server implements CustomTransportStrategy {
       (msg: Record<string, any>) => this.handleMessage(msg, channel),
       {
         noAck,
```
Comment (webjoaoneto) [inline packages/microservices/server/server-rmq.ts:159]: i kept this noAck parameter from queueOptions to maintain the behaviour for all users that use this flag on options.

File: packages/microservices/server/server-rmq.ts (hunk 1 above)
Comment (kamilmysliwiec) [reply packages/microservices/server/server-rmq.ts:47]: So could you please update it?

**Pull request closed without merging**